share/models/validators.py

```diff
@@ -118,7 +118,7 @@ def json_schema_for_field(self, field):
             return rel
         if field.choices:
             return {
-                'enum': field.choices,
+                'enum': [c[0] for c in field.choices],
```

Comment:
Whoops! I know how to django... Wouldn't you want to be validating against label in that case?